requirements/product-environment/product-environment.tex

```diff
@@ -0,0 +1,41 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}  
+      \begin{itemize}
+      \item A connection to a LAN with a central server
+      \item 2 MB of storage space and additional 100MB per image
+      \end{itemize}
+    \subsubsection{Software Requirements}  
+      \begin{itemize}
+      \item None, so far
```

Comment:
User clients need to have Python 3 installed.

Also, I think user clients would also benefit from a Docker installation.
That way they could for instance push Docker container images to a private hub for later/shared use.